frontend/src/components/LoginCard.jsx

```diff
@@ -52,7 +60,11 @@ export default function Login() {
             localStorage.setItem("user-threads", JSON.stringify(data));
             setUser(data);
         } catch (error) {
-            showToast("An error occurred.", error, "error");
+            if (error instanceof z.ZodError) {
+                showToast("An error occurred.", error.errors.map(e => e.message).join(" "), "error");
+            } else {
+                showToast("An error occurred.", error, "error");
+            }   
```

Comment:
Errors message should be shown below input fields
Source: https://www.youtube.com/watch?v=cc_xmawJ8Kg&t=395s